src/NetSuiteClient.php

```diff
@@ -48,6 +49,28 @@ public function __construct($config, $options = array(), $client = null)
         $options = $this->createOptions($this->config, $options);
         $wsdl = $this->createWsdl($this->config);
         $this->client = $client ?: new SoapClient($wsdl, $options);
+        if ($config['host'] == 'https://webservices.netsuite.com') {
+            // Fetch the data center URL for this account because the user
+            // provided the legacy webservices URL.
+            $this->setDataCenterUrl($config);
+        }
+    }
+
+    /**
+     * Set the data center URL for the configured NetSuite account
+     *
+     * @param array $config
+     *
+     * @return void
+     */
+    public function setDataCenterUrl(array $config): void
```

Comment:
void return types are only for PHP 7.1. Do we have to use this here ?

Reply:
Good point, Vinu. I am so accustomed to being able to use current version php at all times that I failed to consider that one. I'll fix it.

Reply:
Awesome @jrebs. this helped us a lot.